SQLQuery_ZooManagement.sql

```diff
@@ -0,0 +1,164 @@
+CREATE Database ZooManagement;
+
+Use ZooManagement;
+
+CREATE TABLE Species
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Species VARCHAR(150),
+SpeciesNumber INT,
+AnimalsNumber INT,
+);
+
+CREATE TABLE Visitors
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Day VARCHAR(150),
+NumberOfVisitors INT,

```

Comment:
In the visitors tale we should store information about the visitors themselves like their name, their age or similar stuff. Then the number of them can be computed based on the information stored in this table so NumberOfVisitors should not be stored.